--- /etc/pam.d/system-auth.dist	2013-09-28 23:23:48.000000000 +0200
+++ /etc/pam.d/system-auth	2013-10-02 21:35:51.998663106 +0200
@@ -1,6 +1,7 @@
 #%PAM-1.0
 
 auth      required  pam_unix.so     try_first_pass nullok
+auth      optional  pam_mount.so
 auth      optional  pam_permit.so
 auth      required  pam_env.so
 
@@ -8,9 +9,11 @@
 account   optional  pam_permit.so
 account   required  pam_time.so
 
+password  optional  pam_mount.so
 password  required  pam_unix.so     try_first_pass nullok sha512 shadow
 password  optional  pam_permit.so
 
+session   optional  pam_mount.so
 session   required  pam_limits.so
 session   required  pam_unix.so
 session   optional  pam_permit.so
